cmd/up.go

```diff
@@ -109,6 +110,29 @@ func InteractivelyCreateConfig() *concourse.Config {
 	dbInstanceClass := AskForRequiredInput("DB Instance Class", AskOptions{Default: "db.t2.micro"})
 	instanceType := AskForRequiredInput("Concourse Instance Type", AskOptions{Default: "t2.micro"})
 
+	possible_elb_protocols := []string{"http", "https"}
```

Comment:
I would like to make variables' names lowerCamelCase instead of lower_snake_case here and there.